x-pack/plugins/reporting/server/config/schema.test.ts

```diff
@@ -9,203 +9,11 @@ import { ConfigSchema } from './schema';
-          "browser": Object {
-            "autoDownload": true,
-            "chromium": Object {
-              "proxy": Object {
-                "enabled": false,
-              },
-            },
-            "type": "chromium",
-          },
-          "loadDelay": "PT3S",
-          "maxAttempts": 1,
-          "networkPolicy": Object {
-            "enabled": true,
-            "rules": Array [
-              Object {
-                "allow": true,
-                "host": undefined,
-                "protocol": "http:",
-              },
-              Object {
-                "allow": true,
-                "host": undefined,
-                "protocol": "https:",
-              },
-              Object {
-                "allow": true,
-                "host": undefined,
-                "protocol": "ws:",
-              },
-              Object {
-                "allow": true,
-                "host": undefined,
-                "protocol": "wss:",
-              },
-              Object {
-                "allow": true,
-                "host": undefined,
-                "protocol": "data:",
-              },
-              Object {
-                "allow": false,
-                "host": undefined,
-                "protocol": undefined,
-              },
-            ],
-          },
-          "timeouts": Object {
-            "openUrl": "PT1M",
-            "renderComplete": "PT30S",
-            "waitForElements": "PT30S",
-          },
-          "zoom": 2,
-        },
-        "csv": Object {
-          "checkForFormulas": true,
-          "enablePanelActionDownload": true,
-          "escapeFormulaValues": false,
-          "maxSizeBytes": ByteSizeValue {
-            "valueInBytes": 10485760,
-          },
-          "scroll": Object {
-            "duration": "30s",
-            "size": 500,
-          },
-          "useByteOrderMarkEncoding": false,
-        },
-        "enabled": true,
-        "encryptionKey": "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
-        "kibanaServer": Object {},
-        "poll": Object {
-          "jobCompletionNotifier": Object {
-            "interval": 10000,
-            "intervalErrorMultiplier": 5,
-          },
-          "jobsRefresh": Object {
-            "interval": 5000,
-            "intervalErrorMultiplier": 5,
-          },
-        },
-        "queue": Object {
-          "indexInterval": "week",
-          "pollEnabled": true,
-          "pollInterval": "PT3S",
-          "pollIntervalErrorMultiplier": 10,
-          "timeout": "PT2M",
-        },
-        "roles": Object {
-          "allow": Array [
-            "reporting_user",
-          ],
-          "enabled": true,
-        },
-      }
-    `);
+    expect(ConfigSchema.validate({}, { dev: false, dist: false })).toMatchSnapshot();
   });
 
   it(`context {"dev":false,"dist":true} produces correct config`, () => {
-    expect(ConfigSchema.validate({}, { dev: false, dist: true })).toMatchInlineSnapshot(`
```

Comment:
There was just too much inline snapshot content in this file and it was affecting syntax highlighting in my editor. So I used file snapshots here instead.